sklearn/tests/test_pipeline.py

```diff
@@ -1071,3 +1071,13 @@ def test_make_pipeline_memory():
     assert pipeline.memory is None
 
     shutil.rmtree(cachedir)
+
+
+def test__len__():
+    scaler_for_pipeline = StandardScaler()
+    km_for_pipeline = KMeans(random_state=0)
+    pipeline = Pipeline([
+        ('scaler', scaler_for_pipeline),
+        ('Kmeans', km_for_pipeline)
+    ])
+    assert pipeline.__len__() == 2
```

Comment:
probably better tested as `assert len(pipeline) == 2`

Reply:
```suggestion
    assert len(pipeline) == 2
```

Reply:
yes, I think I should modify it.